webapp/graphite/metrics_filter.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from django.conf import settings
+from graphite.metric_filter.models import UserMetricFilter
+from graphite.metric_filter.models import GroupMetricFilter
+
+
+def nodeFilter(user, nodePath):
+  if settings.ALLOW_METRICFILTER_BY_USER:
+    if user.id is not None:
+      filters = UserMetricFilter.getFilters(user.id)
+      if filters is not None:
+        for filter in filters.all():
+          matching = match(nodePath, filter)
+          if matching is not None:
+            return matching
+    for filters in GroupMetricFilter.getFiltersByUser(user):
+      if filters is not None:
+        for filter in filters.all():
+          if filter is not None:
+            matching = match(nodePath, filter)
+            if matching is not None:
+              return matching
+    return False
+  else:
+    return True
+
+def match(path, filter):
+  #log.info('testing %s' % filter.filter)
+  pattern = re.compile(filter.filter)
+  if pattern.match(path):
+    if filter.type_filter == 0:
+      return True
+    elif filter.type_filter == 1:
```

Comment:
Would you mind setting these as constants instead of [magic numbers](http://stackoverflow.com/a/47902)? I had to go up to the model reference to know this was include vs exclude. For constants like `INCLUDE_FILTER` or `EXCLUDE_FILTER` I see no reason it would hurt for them to be globals, but up to you.